cypress/e2e/home.cy.ts

```diff
@@ -10,34 +13,76 @@ describe('Visits the home page', () => {
         // TODO: clicking it should take us to '/'
     })
 
+    it('renders the map', () => {
+        // TODO
+        cy.get('[data-testid="map-of-japan"]')
+            .should('have.attr', 'src')
+            .and('not.be.empty')
+    })
+
     it('allows setting search fields', () => {
         cy.get('[data-testid="search-button"]').should('be.visible')
         // TODO
     })
 
-    it.skip('selects a language', () => {
+    it('can select "select a language"', () => {
+        cy.get('[data-testid="search-bar-language"]').trigger('click')
         // TODO
     })
 
-    it.skip('renders the map', () => {
-        // TODO
+    it('can select "English" from the language bar', () => {
+        cy.get('[data-testid="search-bar-language"]').should('be.visible')
+
+        cy.get('[data-testid="search-bar-language"]').select('English')
+
+        cy.get('[data-testid="search-bar-language"]').should('have.value', 'English')
     })
 
-    it.skip('shows doctors nearby', () => {
+    it('can select "Japanese" from the language bar', () => {
+        cy.get('[data-testid="search-bar-language"]').should('be.visible')
+
+        cy.get('[data-testid="search-bar-language"]').select('Japanese')
+
+        cy.get('[data-testid="search-bar-language"]').should('have.value', 'Japanese')
+    })
 
+    it('shows doctors nearby', () => {
+        cy.contains('Doctors Nearby').should('be.visible')
     })
 
-    it.skip('displays the footer', () => {
+    describe('Checks footer links', () => {
+        it('navigates to github', () => {
         // TODO
 
-        // verify link to GitHub
+            // verify link to GitHub
+            cy.get('[data-testid="github-link"]').should('be.visible')
+            cy.get('[data-testid="github-link"]').should('have.attr', 'href', 'https://github.com/ourjapanlife/findadoc-web/')
```

Comment:
Should we make a const for the github url, for consistency?

Reply:
I'll take care of that